src/execution_plan/ops/op_argument_list.c

```diff
@@ -82,7 +88,7 @@ static inline OpBase *ArgumentListClone
 	const OpBase *opBase
 ) {
 	ASSERT(opBase->type == OPType_ARGUMENT_LIST);
-	return NewArgumentListOp(plan);
+	return NewArgumentListOp(plan, opBase->modifies);
```

Comment:
Does `opBase->modifies` is of type `arr.h` ?

Reply:
yes took the same code from op_argument